plugins/chat/spec/system/list_channels/mobile_spec.rb

```diff
@@ -201,6 +201,54 @@
     end
   end
 
+  context "when chat_preferred_mobile_index is set to direct_messages" do
+    before { SiteSetting.chat_preferred_mobile_index = "direct_messages" }
+
+    it "changes the default index" do
+      visit("/chat")
+
+      expect(page).to have_current_path("/chat/direct-messages")
+    end
+
+    context "when user can't use direct messages" do
+      before { SiteSetting.direct_message_enabled_groups_map }
```

Comment:
I think we need to set the preferred mobile index here too, and also assign a value to `direct_message_enabled_groups_map` here that the user isn't part of?

Reply:
yes Im working on fixing this spec